internal/controller/argocd.go

```diff
@@ -239,6 +239,8 @@ func (clctrl *ClusterController) DeployRegistryApplication() error {
 			return err
 		}
 
+		log.Info().Msg("successfull in restarting\n")
```

Comment:
Seems like quite arbitrary and ends with a line break? All log lines include already the line break.